libraries/plugins/webserver/webserver_plugin.cpp

```diff
@@ -251,6 +251,8 @@ void webserver_plugin_impl::handle_http_message( websocket_server_type* server,
    {
       auto body = con->get_request_body();
 
+      con->append_header( "Content-Type", "application/json" );
```

Comment:
This should be done after `con->set_body( api->call( body ) );` because the payload is only JSON after this point.